modules/ROOT/pages/amazon/amazon-sns-connector-examples.adoc

```diff
@@ -0,0 +1,273 @@
+. Go back to the flow and select *Publish*:
+. Configure the remaining parameters:
++
+[%header%autowidth.spread]
+|===
+|Parameter |Value
+2+|*Basic Settings*
+|*Display Name* |`Publish` or any other name you prefer.
+|*Connector configuration*|`Amazon_SNS_configuration` (the reference name to the global element you created).
+2+|*General*
+|*Topic arn* |`arn:aws:sns:us-east-1:480410955647:sns-test` or any other Amazon resource name (ARN).
+|*Message* |``Hello World!``
+|*Subject* |`Testing publish to queue`
+|*Message structure* |`JSON`
+|===
++
+For example:
++
+image::amazon/amazon-sns-publish-message.png[sns-publish-message-props]
++
+. Select the *Configuration XML* tab to view the corresponding XML:
++
+[source,xml]
+----
+<sns:publish doc:name="Publish"
+  config-ref="Amazon_SNS_configuration"
+  topicArn="arn:aws:sns:us-east-1:480410955647:sns-test"
+  messageStructure="JSON">
+  <sns:message ><![CDATA[#[`Hello World`]]]></sns:message>
+</sns:publish>
+----
+
+=== Display the Data Processed by the Publish Operation
+
+In the same same example, add a *Logger* component to display the data processed by the *Publish* operation on the Mule console.
+
+. In the Mule Palette, search for *Logger*, and drag next to the Amazon SNS connector (*Publish*).
+. Configure the *Logger* component as follows:
++
+[%header%autowidth.spread]
+|===
+|Parameter |Value
+|*Display Name* |`Logger` (or any other name you prefer)
+|*Message* |`#[payload]`
+|*Level* |`INFO`
+|===
++
+image::amazon/amazon-sns-logger.png[sns-logger]
+
+[[receive-sns]]
+=== Receive SNS Messages
+
+In the same example, add a second flow that enables Amazon SQS to receive messages published by Amazon SNS:
+
+. In the Mule Palette, search for *Flow* and drag the *Flow* component to the canvas.
+. In the Mule Palette, search for *SQS*, select the *Receivemessages* operation, and drag it to the left of the *Flow* component.
+. Select the new component.
+. Click *+* next to the *Connector configuration* field to add a new Amazon SQS connector global element.
+. Configure the global element as follows:
++
+[%header%autowidth.spread]
+|===
+|Parameter |Description|Value
+|Name |Enter a name for the configuration to reference it. |<Configuration_Name>
+|Access Key |Alphanumeric text string that uniquely identifies the user who owns the account. |Config_Access_Key
+|Secret Key |Key that plays the role of a password. |<Config_Secret_Key>
+|Default Global Queue URL |Set the queue URL |<Config_Queue_URL>.
+|Region Endpoint |Queue region
+|===
++
+For example:
++
+image::amazon/amazon-sns-sqs-config.png[sns-sqs-config]
++
+Select the *Configuration XML* tab to view the corresponding XML:
++
+[source,xml]
+----
+<sqs:config name="Amazon_SQS_Configuration"
+  doc:name="Amazon SQS Configuration"
+  defaultQueueUrl="https://sqs.us-east-1.amazonaws.com/sns-test" >
+<sqs:basic-connection accessKey="AAAJSIIXXJLRPC632UV"
+  secretKey="ZZZo7kc4luuvsCjDnUjftK" />
```

Comment:
Replace accessKey value with AKIAIOSFODNN7EXAMPLE and secretKey value with wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY please.